modules/ROOT/pages/configure-teams.adoc

```diff
@@ -0,0 +1,135 @@
+= Configure Teams
+
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Every root organization has a _root team_ that appears by default. Every user in your organization is automatically a member of this root team and receives all permissions therein.
+You can create _child teams_ under the parent team and configure those teams to have different sets of permissions.
+After you create child teams and assign permissions, you can add users to each of your teams according to what permissions the users should have.
+Further, you can assign _team maintainers_ who can modify users and permissions within the teams they maintain.
+
+Team maintainers can do the following operations on their teams: 
+
+* Change the team name
+* Add or remove members from the team
+* Add or remove additional team maintainers
+* Move the team under different parent teams (if they maintain the other parent team)
+* Delete the team
+
+== Create a Child Team
+
+If you have the Organization Administrator permission in your root organization, you can create child teams under any existing parent teams, including the root team. Users who have the Organization Administrator permission in business groups can create child teams under the root team and then become team maintainers for that team. Additionally, team maintainers can create child teams under teams that they maintain.
+The child team inherits permissions from its parent team, but you configure additional permissions for its users.
+
+To create a child team in the Teams feature interface:
+
+include::partial$include-nav-steps-org-admin.adoc[]
+
+include::partial$include-nav-teams.adoc[]
+
+. Select *Create Team*.
+. Enter a unique and intuitive name for your child team. +
+You can use only alphanumeric characters, hyphens, commas, periods, and spaces. The character limit is 255.
++
+. Select a parent team.
+. Select *Save*.
+
+The child team appears under its parent team in the list of teams. You can also select a parent team and navigate to the *Child teams* tab to see a list of all child teams under that parent team.
+
+== Manage Team Permissions
+
+When you begin using the Teams feature, your organization’s main team is named “Everyone at <Your Organization>” and does not have any permissions assigned.
+If you have the Organization Administrator permission, you can configure permissions for this team in addition to child teams.
+If you invite users to “Everyone at <Your Organization>” without assigning any permissions to that team, your users cannot execute tasks or view assets in your organization.
+
+If you have the Organization Administrator permission in a specific business group but not your entire organization, you can assign these permissions to any of existing teams or users within your organization.
```

Comment:
One more: this sentence refers to OA for business group but not org, whereas earlier text refers to OA for root and business group. So is this hierarchy, akin to teams, root > org > business group > teams > users?

Reply:
"these permissions" = which permissions?

Reply:
```suggestion
If you have the Organization Administrator permission in a specific business group but not your entire organization, you can assign permissions to any existing teams or users within your organization.
```
"existing" makes me wonder: have we addressed how, besides inheritance, additional teams or users are given permissions? See Line 82 comment.